lupa/lua.pxd

```diff
@@ -439,14 +439,15 @@ cdef extern from *:
     #define lua_objlen(L, i) lua_rawlen(L, (i))
     #endif
 
-    #if LUA_VERSION_NUM >= 504
-    #define read_lua_version(L)  ((int) lua_version(L))
-    #elif LUA_VERSION_NUM >= 502
-    #define read_lua_version(L)  ((int) *lua_version(L))
-    #elif LUA_VERSION_NUM >= 501
```

Comment:
Why did you remove this? It is meant to read the runtime version of Lua that is used, not the compile-time version.

Reply:
You're right. This is not necessary to change this for this specific PR.